File: apps/auth-service/src/main/java/com/booster/authservice/web/dto/TokenResponse.java
package com.booster.authservice.web.dto;

public record TokenResponse(
        String accessToken,
        String type,
        long expiresIn
) {
    public static TokenResponse of(String accessToken, long expiresIn) {
        return new TokenResponse(accessToken, "Bearer", expiresIn);
    }
}

File: apps/kotlin/shopping-service/src/main/kotlin/com/booster/kotlin/shoppingservice/auth/web/dto/response/TokenResponse.kt
package com.booster.kotlin.shoppingservice.auth.web.dto.response

import com.booster.kotlin.shoppingservice.auth.application.dto.TokenResult

data class TokenResponse(
    val accessToken: String,
    val refreshToken: String,
) {
    companion object {
        fun from(result: TokenResult) = TokenResponse(
            accessToken = result.accessToken,
            refreshToken = result.refreshToken,
        )
    }
}
